server/styles/Home.module.css

```diff
@@ -0,0 +1,116 @@
+.container {
```

Comment:
This is all auto-generated and is part of the next app that will be removed before merging.  Ignore

Reply:
> This is all auto-generated and is part of the next app that will be removed before merging. Ignore

It's now removed